package.json

```diff
@@ -33,70 +33,85 @@
     "test": "tests"
   },
   "scripts": {
-    "build": "ember build",
+    "build": "ember build --environment=production",
+    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel 'lint:!(fix)'",
+    "lint:fix": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*:fix",
     "lint:hbs": "ember-template-lint .",
-    "lint:js": "eslint .",
+    "lint:hbs:fix": "ember-template-lint . --fix",
+    "lint:js": "eslint . --cache",
+    "lint:js:fix": "eslint . --fix",
     "start": "ember serve",
-    "test": "ember test",
-    "test:all": "ember try:each",
+    "test": "npm-run-all lint test:*",
+    "test:ember": "ember test",
+    "test:ember-compatibility": "ember try:each",
     "postversion": "git push && git push --tags"
   },
   "resolutions": {
-    "ember-cli-template-lint/**/ember-template-lint": "^1.4.0",
     "ember-data": "3.5.2",
     "whatwg-fetch": "3.3.1"
```

Comment:
Could we trying dropping these in this PR. I think it should be safe. They were only added in #431

Reply:
Yup, looks like it!